client/tests/integration/queries/account.rs

```diff
@@ -66,8 +67,8 @@ fn find_accounts_with_asset() -> Result<()> {
     );
 
     let found_accounts = test_client
-        .request(client::account::all_with_asset(definition_id))?
-        .collect::<QueryResult<Vec<_>>>()?;
+        .query(client::account::all_with_asset(definition_id))
+        .execute_all()?;
```

Comment:
```suggestion
        .execute()?;
```

not sure if I mentioned this already, but also this should be the default IMO

Reply:
or how about just:
```suggestion
        .all()?;
```

Reply:
wait, I'll collect my remarks separately